internal/installation/paths_windows.go

```diff
@@ -3,10 +3,23 @@ package installation
 import (
 	"os"
 	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
 )
 
 func installPathForBranch(branch string) (string, error) {
-	return filepath.Join(os.Getenv("USERPROFILE"), "AppData", "Local", "ActiveState", "StateTool", branch), nil
+	home := os.Getenv("USERPROFILE")
+	if dir := os.Getenv(constants.HomeEnvVarName); dir != "" {
+		home = dir
+	}
```

Comment:
Please refactor this to use the `internal/osutils/user` package's `HomeDir()` function to get the user's home directory.